scripts/standalone.mk

```diff
@@ -43,6 +43,12 @@ LINK_TARGET = ramrodata
 endif
 endif
 
+ifeq ($(PROGRAM),coremark)
+ifeq ($(LINK_TARGET),)
+LINK_TARGET = ramrodata
```

Comment:
Does this fit in the 16k of ram on the freedom-e310-arty and sifive-hifive1*? If it doesn't we need to disable this option for those targets.

Reply:
Thanks for reminding. Yes, I encountered the itim size too small problem with CONFIGURATION=debug. To get the best coremark score, users should set CONFIGURATION to  "release". So, I've updated the lds selection~

Reply:
That technically fixes the problem, but what I actually mean is this: https://github.com/sifive/freedom-e-sdk/blob/development-19.05/scripts/standalone.mk#L44

Reply:
For freedom-e310 and hifive1 series, it fits in the 16k of ram.
  text   data   bss     dec    hex   filename
28346 6784 5330  40460 9e0c ./coremark.elf

Reply:
Awesome, perfect.